### ISSUE TITLE
About the calibration data set

### ISSUE THREAD

**Issue by Eduiskss** [ModelCloud/GPTQModel#1483]
Can you tell me how to prepare the calibration dataset for gptq, and whether it can be ideally the same as the training dataset, I would like to quantize the 7B model after fine-tuning, and I don't know how to prepare the calibration dataset

**Comment by Qubitium** [comment]
@Eduiskss  You can use a subset of your training set for calibration dataset. Good to have varied subset that covers all down-stream tasks.

**Comment by Eduiskss** [comment]
> [@Eduiskss](https://github.com/Eduiskss) You can use a subset of your training set for calibration dataset. Good to have varied subset that covers all down-stream tasks.

Thank you very much, I would like to ask for what is the appropriate number of subsets, what percent of the training set

**Comment by Qubitium** [comment]
@Eduiskss Just enough so that the loss column is very small. Try it. There is no exact science to say how many rows you need. Sometimes more, sometimes less. Start with 512 rows of data and try multiple of that.
